### PR TITLE
convert onwards/tonal to es6

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

converts tonal module to es6

## What is the value of this and can you measure success?

:sparkles: more es6 :sparkles:

## Does this affect other platforms - Amp, Apps, etc?

No

## Screenshots

N/A

## Tested in CODE?

No

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/onward/tonal.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        mediator.emit('ui:images:upgradePictures');
+        mediator.emit('modules:tonal:loaded');
+        register.end('tonal-content');
+    }
+
+    static error(): void {
+        register.error('tonal-content');
+    }
+
+    constructor(): void {
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/onward/tonal.js]: Could you move the constructor to the beginning of the class (just for readability)?

File: static/src/javascripts/projects/common/modules/onward/tonal.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        // Ensures we only fetch supported tones.
+        if (this.isSupported()) {
+            this.endpoint = this.getEndpoint();
+        } else {
+            this.fetch = noop;
+        }
+    }
+
+    getEndpoint(): string {
+        return `/container/${tones[this.edition][TonalComponent.getTone()]}.json`;
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/onward/tonal.js]: For readability: could you save `tones[this.edition][TonalComponent.getTone()]` as a variable before?

**Review by gu-stav — APPROVED** [review]
Just 💅 comments.

**Review by SiAdcock — APPROVED** [review]

**Comment by gustavpursche** [comment]
I've just updated `prettier`, which takes care of the code formatting. Could you please rebase/ merge master, before merging this in, to avoid broken checks?

Steps:
- Merge master
- Run `make fix` and commit in case there are changes

Thanks 🙏

File: static/src/javascripts/projects/common/modules/onward/tonal.js (hunk 1 above)
Comment (GHaberis) [inline static/src/javascripts/projects/common/modules/onward/tonal.js]: The JS validation enforced static properties at the beginning of the class.

**Pull request merged**